src/components/App.js

```diff
@@ -16,25 +17,27 @@ export default function App() {
   );
   const [config, setConfig] = useState(
     localStorage.getItem("linkr")
-      ? { headers: { Authorization: `Bearer ${userData.token}`}}
+      ? { headers: { Authorization: `Bearer ${userData.token}` } }
       : null
```

Comment:
**Suggestion:** this logic can be transformed and isolated in a helper, so the app file doesn't get dirty

Reply:
Perfect, we'll do it today